homeassistant/components/lamarzocco/coordinator.py

```diff
@@ -94,3 +94,8 @@ async def _async_handle_request(
         except RequestNotSuccessful as ex:
             _LOGGER.debug(ex, exc_info=True)
             raise UpdateFailed("Querying API failed. Error: %s" % ex) from ex
+
+    @property
+    def local_connection_set(self) -> bool:
```

Comment:
```suggestion
    def has_local_connection(self) -> bool:
```
Maybe sounds better

Reply:
oh and by the way, since this field never changes, we can move this in the constructor. That's a bit more efficient